playbooks/tests/tasks/rally/run.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+source /root/stackrc
+
+BUILD_TAG=${1:-master}
+RALLY=~/rally/bin/rally
+RALLY_INSTALL_URL=https://raw.githubusercontent.com/openstack/rally/master/install_rally.sh
+RALLY_FILE=bbc-cloud-validate
+RALLY_TEST_IMAGE="cirros"
+RALLY_TEST_NET_ID=$(openstack network show internal -c id -f value)
+RALLY_TEST_VCPU_LIMIT=4
+
+read -r -d '' RALLY_SED_STRING <<EOM
+s/@@NET_ID@@/${RALLY_TEST_NET_ID}/g;
+s/@@IMAGE_NAME@@/${RALLY_TEST_IMAGE}/g;
+s/@@VCPU_LIMIT@@/${RALLY_TEST_VCPU_LIMIT}/g
+EOM
+
+sed -e "${RALLY_SED_STRING}" ${RALLY_FILE}.yml.tmpl > ${RALLY_FILE}.yml
```

Comment:
This shouldn't be necessary as we can write the data at run time to be used in the task template. Rally supports this natively we shouldn't have to sed stuff.